go/cmd/dolt/commands/init_test.go

```diff
@@ -96,3 +101,37 @@ func TestInitTwice(t *testing.T) {
 		t.Error("Second init should fail")
 	}
 }
+
+// testLocalConfigValue tests that local config data is set correctly when the specified argument
+// is present in the command line args, and is not set when the argument is not present.
+func testLocalConfigValue(t *testing.T, dEnv *env.DoltEnv, test initTest, argKey, envKey string) {
+	localConfig, ok := dEnv.Config.GetConfig(env.LocalConfig)
+	if !ok {
+		t.Error(test.Name, "- Unable to load local configuration")
+		return
+	}
+
+	found := false
+	expectedValue := ""
+	for i := 0; i <= len(test.Args)-2; i = i + 2 {
+		if test.Args[i] == "-"+argKey {
+			found = true
+			expectedValue = test.Args[i+1]
+		}
+	}
+
+	actualValue, err := localConfig.GetString(envKey)
+	if found {
+		if err != nil {
```

Comment:
Nit: try not to write any new tests without testify

Reply:
ooooh... good call. I migrated this unit test over to stretchr/testify and it's cleaner/leaner now.